config/crd/patches/webhook_in_ansibleTests.yaml

```diff
@@ -0,0 +1,16 @@
+# The following patch enables a conversion webhook for the CRD
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: ansibleTest.test.openstack.org
```

Comment:
@eshulman2 I think we have to go through the occurrences of the `ansibleTest` in the PR. And think whether it should be capitalized or not. As an example you can take a look at HorizonTest CR. You can see that for example here [1] the name for HorionTests CR is not capitalized. 

Also, checking whether the test-operator works from locally build test-operator-index would be in this case a good idea I think [2] (question #7).

[1] https://github.com/openstack-k8s-operators/test-operator/blob/b68150cf867b538bc9896b9d2102d3c29ded2c29/config/crd/patches/webhook_in_horizontests.yaml#L5
[2] https://openstack-k8s-operators.github.io/test-operator/FAQ.html